lms/envs/common.py

```diff
@@ -2004,6 +2004,9 @@
     'openedx.core.djangoapps.self_paced',
 
     'sorl.thumbnail',
+
+    # Needed whether or not enabled, due to migrations
+    'badges',
```

Comment:
I'm not sure whether these should be added as `lms.djangoapps.badges` now.  The list seems to be inconsistent, so up to you what to do.